local03-showrspnotice.sh

```diff
@@ -1,22 +1,43 @@
 #!/bin/sh
 
+site_recommendation () {
+    cat << EOF
+
+To create a Rubin Observatory environment in a terminal session and set up
+an appropriate set of packages:
+
+EOF
+
+    case $RSP_SITE_TYPE in
+        telescope | staff)
+            echo "        setup lsst_sitcom"
+	    echo ""
+            ;;
+        *)
+            echo "        setup lsst_distrib"
+	    echo ""
+            ;;
+    esac
+}
```

Comment:
Did you check whether this is needed at all? Since it's done by lsstlaunch.bash, I suspect it is not.